modules/internal/OwnedObject.chpl

```diff
@@ -193,9 +197,9 @@ module OwnedObject {
       If the argument is `nil` it returns `nil`.
 
       .. note::
-
-         This is part of an experimental interface, users should prefer
-         :proc:`owned.create`/:proc:`owned.retain`/:proc:`owned.clear` for now.
+         This is part of an new interface that will replace :proc:`owned.clear`.However, `release` is not as widely used as `clear` yet, so there may
```

Comment:
This line looks accidentally too long

Reply:
oops I was too aggressive in removing whitespace. fixed